src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearchindexer/integration/wiremock/RestrictedPatientApiMockServer.kt

```diff
@@ -2,12 +2,13 @@ package uk.gov.justice.digital.hmpps.prisonersearchindexer.integration.wiremock
 
 import com.github.tomakehurst.wiremock.WireMockServer
 import com.github.tomakehurst.wiremock.client.WireMock
+import com.github.tomakehurst.wiremock.common.ClasspathFileSource
 import org.junit.jupiter.api.extension.AfterAllCallback
 import org.junit.jupiter.api.extension.BeforeAllCallback
 import org.junit.jupiter.api.extension.BeforeEachCallback
 import org.junit.jupiter.api.extension.ExtensionContext
 
-class RestrictedPatientApiMockServer : WireMockServer(8095) {
+class RestrictedPatientApiMockServer : WireMockServer(8095, ClasspathFileSource("restricted-patients"), false) {
```

Comment:
I've copied across all the mappings from the prisoner offender search project to here.  Would be good to get feedback on this approach - do people prefer mappings in code rather than json?

Reply:
I mush prefer mappings in code. But that is personal to me. I find json resource directories confusing and snuggle to find them. Maybe do a poll in on Slack